dts/arm/96b_carbon.dts

```diff
@@ -8,8 +8,9 @@
 #include <st/stm32f401.dtsi>
 
 / {
-	model = "STMicroelectronics STM32F401RE-NUCLEO board";
-	compatible = "st,stm32f401re-nucleo", "st,stm32f401";
+	model = "Seeed Studio Carbon 96board";
+	compatible = "seeed,carbon", "st,stm32f401re-nucleo",
```

Comment:
@mbolivar,  for now we've been trying to adapt from what was done on Linux. But I agree a real policy is missing.
In the long range, we target to have common dts files Linux/Zephyr/.. , so we need to align with ST guys working on Linux dts files for STM32 and come up with a real policy. Will keep you posted.